app/assets/javascripts/inspections.js.coffee

```diff
@@ -0,0 +1,20 @@
+jQuery ->
```

Comment:
Good catch.  This should not be in both inspections.js.coffee and items.js.coffee.